src/styles/blocks/main-container.scss

```
@@ -1,15 +1,14 @@
 .main-container {
-  display: flex;
-  justify-content: center;
+  margin: 0 auto;
 
   &__wrap {
     width: catalogWidth($cardWidth, $cardMargin, $columns);
-    display: flex;
-    flex-wrap: wrap;
-    margin: 50px auto;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    margin: 0 auto;
```

```suggestion
    display: grid;
    grid-template-columns: repeat(4, 200px);
    margin: 0 auto;
```
замість 1fr, варто задати ширину карточки 200px. 
ще можна властивістю 
gap задати відступи між карточками

---

Tetiana is correct. It's better to keep column width same as card width and add spaces between cards using grid gaps.

---

пробував зробити з gap, але тести його чомусь не пропускають. довелось залишити з марджіном